server/collaborator/src/account.ts

```diff
@@ -27,7 +27,7 @@ export async function getWorkspaceInfo (token: string): Promise<ClientWorkspaceI
       },
       body: JSON.stringify({
         method: 'getWorkspaceInfo',
-        params: []
+        params: [true] // Collaborator should update last used time.
```

Comment:
Not sure about it

Reply:
Let it be, probable we need to update lastVisit if person has active connection to workspace.